core-web/libs/edit-content-bridge/jest.config.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+export default {
+    displayName: 'edit-content-bridge',
+    preset: '../../jest.preset.js',
+    transform: {
+        '^.+\\.[tj]s$': ['ts-jest', { tsconfig: '<rootDir>/tsconfig.spec.json' }]
+    },
+    moduleFileExtensions: ['ts', 'js', 'html'],
+    coverageDirectory: '../../coverage/libs/edit-content-bridge'
```

Comment:
@oidacra, I think it is not necessary because the `coverageDirectory` is already defined as `/target/core-web-reports/ `in the `jest.preset.js` file.

Reply:
removed.